federation/cmd/federation-apiserver/app/options/validation.go

```diff
@@ -27,6 +29,18 @@ func (options *ServerRunOptions) Validate() []error {
 	if errs := options.InsecureServing.Validate("insecure-port"); len(errs) > 0 {
 		errors = append(errors, errs...)
 	}
+	if errs := options.Authentication.Validate(); len(errs) > 0 {
+		errors = append(errors, errs...)
+	}
+	if errs := options.Authorization.Validate(); len(errs) > 0 {
+		errors = append(errors, errs...)
+	}
+	if errs := options.CloudProvider.Validate(); len(errs) > 0 {
+		errors = append(errors, errs...)
+	}
+	if options.EventTTL <= 0 {
```

Comment:
oops, I replied in outside context as the input box here is a bit small to reply.